.github/actions/check_for_stc/action.yml

```diff
@@ -0,0 +1,13 @@
+name: 'Ensure STC is installed with Pyccel'
+
+runs:
+  using: "composite"
+  steps:
+    - name: check stc installation
+      shell: python
+      run: |
+        from extensions import STC
+        from os.path import exists, join
+        pyccel_loc = stc.__path__[0]
+        print(pyccel_loc)
+        assert(exists(join(pyccel_loc,'include')))
```

Comment:
You need to ensure that this action is called from a workflow. I think it makes sense in the workflows which check the wheels

Reply:
Sorry. I'll leave you to it and only look when you request a review ;)